### ISSUE TITLE
Screen reader does not announce the content and buttons

### ISSUE THREAD

**Issue by veikyg** [tinymce/tinymce#10593]
When a screen reader such as NVDA/JAWS is used, keyboard shortcuts work to move focus to various sections such as Toolbar, etc. But no announcements are made. Accessible labels of the controls is not announced by the screen reader.

### ✔️ Expected result
Accessible labels (aria-label) of the controls should be announced when focused.

### ❌ Actual result
No announcement is made

### ❓ Possible solution
-

## 📃 Other details
- Browser: … Chrome Latest
- OS: Windows 11
- First affected version: …
- Worked in version: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by leonardcoutinho** [comment]
I have same problem here, NVDA can't read tinymce on Chrome
